src/resources/js/views/rsvp.js

```diff
@@ -0,0 +1,183 @@
+(function( $, my ) {
+	'use strict';
+
+	// base elements
+	var tribe_rsvp = $( '.tribe-block__rsvp' );
+
+	// Bail if we don't have what we need
+	if ( 0 === tribe_rsvp.length ) {
```

Comment:
Could simplify and just make `! tribe_rsvp.length`

Reply:
Yes I think that one would work in this case was a `cp` operation from `extension` as this files were not part of the merge but I think we should do a bit of review on them afterwards 👍